packages/paste-core/components/file-picker/src/FilePicker.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Text} from '@twilio-paste/text';
+import {useUID} from '@twilio-paste/uid-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {SiblingBox} from '@twilio-paste/sibling-box';
+
+export interface FilePickerProps extends React.HTMLAttributes<HTMLInputElement>, Pick<BoxProps, 'element'> {
+  accept?: string;
+  children: React.ReactElement;
+  disabled?: boolean;
+  i18nNoSelectionText?: string;
+  required?: boolean;
+}
+
+const FilePicker = React.forwardRef<HTMLInputElement, FilePickerProps>(
+  (
+    {
+      element = 'FILEPICKER',
+      accept,
+      id = useUID(),
+      children,
+      disabled = false,
+      i18nNoSelectionText = 'No files selected',
```

Comment:
suggestion: since we can only choose one file at a time, I think this should match that by being singular
```suggestion
      i18nNoSelectionText = 'No file selected',
```

Reply:
agreed :) talked to Aren about language and we're going with "No file uploaded" and "Upload a file"